tests/test_json_formatter.py

```diff
@@ -79,6 +83,89 @@ def test_simple_log(self):
             ])
         )
 
+    @params(
```

Comment:
Probably add one param set, where no `time` is used?

Reply:
:+1:

Reply:
:+1: